.azuredevops/pipelines/build-and-release.yaml

```diff
@@ -47,73 +47,51 @@ stages:
           - name: Configuration
             value: Release
 
-          - name: ProjectFolder
-            value: src/Kentico.Xperience.Lucene
-
-          - name: ProjectFilePath
-            value: src/Kentico.Xperience.Lucene/Kentico.Xperience.Lucene.csproj
+          - name: slnPath
+            value: src/Kentico.Xperience.Lucene.Libs.sln
 
         steps:
-          - task: PowerShell@2
```

Comment:
I would rather leave the Node setup steps here, we need Node.js to build Admin/Client.